src/board-explorer-app/board-explorer-app.html

```diff
@@ -1429,7 +1431,7 @@
       if (uri[1]) {
         var tokens = uri[1].split(",");
         for (var i = 0; i < tokens.length; i++) {
-          tokens[i] = tokens[i].toUpperCase();
+          tokens[i] = tokens[i];
```

Comment:
We need all the selection to be case insenstive; the board-explorer and board-viewer need to consistently do it for all inbound data (whether via URLs or via the board.json file) so that the string compares are all comparing against the same case.

That was why the `toUpperCase()` was there; it's a user supplied string.

There must be a code path that's bringing data from the board.json that isn't `toUpperCase()`ing perhaps?